Localize/lang/strings.json

```
@@ -264,12 +257,7 @@
   "ALImBL": "(UTC+00:00) Monrovia, Reykjavik",
   "ANbEeQ": "Required. The name of the action whose values you want.",
```

An action isn't a "who":

```suggestion
  "ANbEeQ": "Required. The name of the action that has the values you want.",
```